cmd/abc/abc.go

```diff
@@ -28,8 +28,10 @@ import (
 
 	"golang.org/x/sys/unix"
 
+	"github.com/abcxyz/abc-updater/pkg/metrics"
 	"github.com/abcxyz/abc-updater/pkg/updater"
 	"github.com/abcxyz/abc/internal/version"
+	"github.com/abcxyz/abc/internal/wrapper"
```

Comment:
The name "wrapper" could be more helpful; all it says is that something is being wrapped but not what it is

Reply:
Wasn't sure what to call it, since "metrics" would be a bit confusing (would need to alias one of the imports anyways).

Figured other wrappers could be put in the same package. Open to naming suggestions.

Reply:
How about "metricswrap"? Let's plan on putting each future wrapper in its own package; it's nice to have meaningful package names.